test/Index/property_wrappers.swift

```diff
@@ -18,25 +18,38 @@ var globalInt: Int { return 17 }
 public struct HasWrappers {
   @Wrapper
   public var x: Int = globalInt
+  // CHECK: [[@LINE-2]]:4 | struct/Swift | Wrapper | s:14swift_ide_test7WrapperV | Ref | rel: 0
+  // CHECK-NOT: [[@LINE-2]]:23 | variable/Swift | globalInt | s:14swift_ide_test9globalIntSivp | Ref,Read | rel: 0
```

Comment:
I suggest dropping everything after the name `globalInt` to make this CHECK-NOT more robust against changes.